prow/config.yaml

```diff
@@ -63,6 +63,41 @@ tide:
 
 presubmits:
   tektoncd/plumbing:
+  - name: pull-dogfooding-mario-test
+    agent: kubernetes
+    always_run: true
+    decorate: true
+    rerun_command: "/test pull-dogfooding-mario-test"
+    trigger: "(?m)^/test (all|pull-dogfooding-mario-test),?(\\s+|$)"
```

Comment:
same here, I would not include "all" here - otherwise this will run on all plumbing patches